google/resumable_media/_helpers.py

```diff
@@ -313,7 +317,8 @@ def _parse_checksum_header(header_value, response, checksum_label):
     matches = []
     for checksum in header_value.split(u","):
         name, value = checksum.split(u"=", 1)
-        if name == checksum_label:
+        # Official docs say "," is the separator, but real-world responses have encountered ", "
+        if name.lstrip() == checksum_label:
```

Comment:
If this is true we should add it to the sync version, maybe urgently.

Reply:
I think this is added via a recent PR, https://github.com/googleapis/google-resumable-media-python/pull/170